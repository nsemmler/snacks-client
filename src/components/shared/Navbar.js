import {Navbar, NavItem, Nav, NavLink, NavbarBrand} from 'reactstrap';
import React from 'react';

const NavComponent = () => {
  return (<Navbar>
    <NavbarBrand>Galvanize Snacks</NavbarBrand>
    <Nav className="ml-auto">
      <NavItem>
        <NavLink href="/">Home</NavLink>
      </NavItem>
      <NavItem>
        <NavLink href="/snacks">Snacks</NavLink>
      </NavItem>
    </Nav>
  </Navbar>)
}

export default NavComponent
