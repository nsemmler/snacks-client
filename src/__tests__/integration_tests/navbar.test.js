import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import Navbar from '../../components/shared/Navbar'

describe('Navbar Component', () => {
  test('renders consistently', () => {
    const form = renderer.create(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>
    ).toJSON()
    expect(form).toMatchSnapshot()
  })
})
