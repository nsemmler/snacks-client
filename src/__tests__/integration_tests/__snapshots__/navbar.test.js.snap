// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders consistently 1`] = `
<nav
  className="navbar"
>
  <a
    className="navbar-brand"
  >
    Galvanize Snacks
  </a>
  <ul
    className="ml-auto nav"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/snacks"
        onClick={[Function]}
      >
        Snacks
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Navbar Component renders consistently 1`] = `
<nav
  className="navbar"
>
  <a
    className="navbar-brand"
  >
    Galvanize Snacks
  </a>
  <ul
    className="ml-auto nav"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/snacks"
        onClick={[Function]}
      >
        Snacks
      </a>
    </li>
  </ul>
</nav>
`;
